packages/calcite-components/src/components/block-section/block-section.scss

```diff
@@ -127,3 +125,18 @@ calcite-switch {
 }
 
 @include base-component();
```

Comment:
Can you ensure trailing includes remain at the end? Applies to similarly-changed component styles.